### DISCUSSION TITLE
Using Jquery version 3.5.1 and migrating to bootstrap version 3.4.1 are there any breaking changes in this?

### DISCUSSION THREAD

**Discussion by saybee** [twbs/bootstrap#31450] [General] [answered]
Are both the versions compatible ? or Bootstrap can be upgraded without considering jquery version as a dependency ?
This issue worries me - https://github.com/twbs/bootstrap/issues/30644

**Comment by XhmikosR** [comment, marked as answer]
That issue has since been fixed in v4 and in jQuery 3.5.1
